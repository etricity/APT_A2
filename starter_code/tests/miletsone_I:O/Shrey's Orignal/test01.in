2
loadCase
place O4 at E3
place Y4 at F4
place R4 at G5
save
test01
quit
