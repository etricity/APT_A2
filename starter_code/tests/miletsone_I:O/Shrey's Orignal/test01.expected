Welcome to Qwirkle!
-------------------


Menu
----
1. New Game
2. Load Game
3. Show student infomation
4. Quit

>2
Please enter a filename
>loadCase
Loading Game....
Loading Players...
Loading Board...
Loading bag...
Loading current player...
Game Successfully Loaded 
JACK SWALLOW'S turn.
SHREY PAREKH score: 4
JACK SWALLOW score: 2
  0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
  -----------------------------------------------------------------------------
A |     |     |     |     |     |     |     |     |     |     |     |     |    
B B4 |     |     |     |     |     |     |     |     |     |     |     |     | 
C |  G4 |     |     |     |     |     |     |     |     |     |     |     |    
D    |  P4 |     |     |     |     |     |     |     |     |     |     |     | 
E |     |     |     |     |     |     |     |     |     |     |     |     |    
F    |     |     |     |     |     |     |     |     |     |     |     |     | 
G |     |     |     |     |     |     |     |     |     |     |     |     |    
H    |     |     |     |     |     |     |     |     |     |     |     |     | 
I |     |     |     |     |     |     |     |     |     |     |     |     |    
J    |     |     |     |     |     |     |     |     |     |     |     |     | 
K |     |     |     |     |     |     |     |     |     |     |     |     |    
L    |     |     |     |     |     |     |     |     |     |     |     |     | 
M |     |     |     |     |     |     |     |     |     |     |     |     |    
N    |     |     |     |     |     |     |     |     |     |     |     |     | 
O |     |     |     |     |     |     |     |     |     |     |     |     |    
P    |     |     |     |     |     |     |     |     |     |     |     |     | 
Q |     |     |     |     |     |     |     |     |     |     |     |     |    
R    |     |     |     |     |     |     |     |     |     |     |     |     | 
S |     |     |     |     |     |     |     |     |     |     |     |     |    
T    |     |     |     |     |     |     |     |     |     |     |     |     | 
U |     |     |     |     |     |     |     |     |     |     |     |     |    
V    |     |     |     |     |     |     |     |     |     |     |     |     | 
W |     |     |     |     |     |     |     |     |     |     |     |     |    
X    |     |     |     |     |     |     |     |     |     |     |     |     | 
Y |     |     |     |     |     |     |     |     |     |     |     |     |    
Z    |     |     |     |     |     |     |     |     |     |     |     |     | 
  -----------------------------------------------------------------------------


Your hand is...
Bag Size: 6
P3, P2, G6, O4, B6, G2
> place O4 at E3

SHREY PAREKH's turn.
SHREY PAREKH score: 4
JACK SWALLOW score: 6
  0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
  -----------------------------------------------------------------------------
A |     |     |     |     |     |     |     |     |     |     |     |     |    
B B4 |     |     |     |     |     |     |     |     |     |     |     |     | 
C |  G4 |     |     |     |     |     |     |     |     |     |     |     |    
D    |  P4 |     |     |     |     |     |     |     |     |     |     |     | 
E |     |  O4 |     |     |     |     |     |     |     |     |     |     |    
F    |     |     |     |     |     |     |     |     |     |     |     |     | 
G |     |     |     |     |     |     |     |     |     |     |     |     |    
H    |     |     |     |     |     |     |     |     |     |     |     |     | 
I |     |     |     |     |     |     |     |     |     |     |     |     |    
J    |     |     |     |     |     |     |     |     |     |     |     |     | 
K |     |     |     |     |     |     |     |     |     |     |     |     |    
L    |     |     |     |     |     |     |     |     |     |     |     |     | 
M |     |     |     |     |     |     |     |     |     |     |     |     |    
N    |     |     |     |     |     |     |     |     |     |     |     |     | 
O |     |     |     |     |     |     |     |     |     |     |     |     |    
P    |     |     |     |     |     |     |     |     |     |     |     |     | 
Q |     |     |     |     |     |     |     |     |     |     |     |     |    
R    |     |     |     |     |     |     |     |     |     |     |     |     | 
S |     |     |     |     |     |     |     |     |     |     |     |     |    
T    |     |     |     |     |     |     |     |     |     |     |     |     | 
U |     |     |     |     |     |     |     |     |     |     |     |     |    
V    |     |     |     |     |     |     |     |     |     |     |     |     | 
W |     |     |     |     |     |     |     |     |     |     |     |     |    
X    |     |     |     |     |     |     |     |     |     |     |     |     | 
Y |     |     |     |     |     |     |     |     |     |     |     |     |    
Z    |     |     |     |     |     |     |     |     |     |     |     |     | 
  -----------------------------------------------------------------------------


Your hand is...
Bag Size: 6
R6, R5, R2, Y6, O3, Y4
> place Y4 at F4

JACK SWALLOW'S turn.
SHREY PAREKH score: 9
JACK SWALLOW score: 6
  0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
  -----------------------------------------------------------------------------
A |     |     |     |     |     |     |     |     |     |     |     |     |    
B B4 |     |     |     |     |     |     |     |     |     |     |     |     | 
C |  G4 |     |     |     |     |     |     |     |     |     |     |     |    
D    |  P4 |     |     |     |     |     |     |     |     |     |     |     | 
E |     |  O4 |     |     |     |     |     |     |     |     |     |     |    
F    |     |  Y4 |     |     |     |     |     |     |     |     |     |     | 
G |     |     |     |     |     |     |     |     |     |     |     |     |    
H    |     |     |     |     |     |     |     |     |     |     |     |     | 
I |     |     |     |     |     |     |     |     |     |     |     |     |    
J    |     |     |     |     |     |     |     |     |     |     |     |     | 
K |     |     |     |     |     |     |     |     |     |     |     |     |    
L    |     |     |     |     |     |     |     |     |     |     |     |     | 
M |     |     |     |     |     |     |     |     |     |     |     |     |    
N    |     |     |     |     |     |     |     |     |     |     |     |     | 
O |     |     |     |     |     |     |     |     |     |     |     |     |    
P    |     |     |     |     |     |     |     |     |     |     |     |     | 
Q |     |     |     |     |     |     |     |     |     |     |     |     |    
R    |     |     |     |     |     |     |     |     |     |     |     |     | 
S |     |     |     |     |     |     |     |     |     |     |     |     |    
T    |     |     |     |     |     |     |     |     |     |     |     |     | 
U |     |     |     |     |     |     |     |     |     |     |     |     |    
V    |     |     |     |     |     |     |     |     |     |     |     |     | 
W |     |     |     |     |     |     |     |     |     |     |     |     |    
X    |     |     |     |     |     |     |     |     |     |     |     |     | 
Y |     |     |     |     |     |     |     |     |     |     |     |     |    
Z    |     |     |     |     |     |     |     |     |     |     |     |     | 
  -----------------------------------------------------------------------------


Your hand is...
Bag Size: 6
P3, P2, G6, B6, G2, R4
> place R4 at G5

QWIRKLE!!!

SHREY PAREKH'S turn.
SHREY PAREKH score: 9
JACK SWALLOW score: 18
  0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
  -----------------------------------------------------------------------------
A |     |     |     |     |     |     |     |     |     |     |     |     |    
B B4 |     |     |     |     |     |     |     |     |     |     |     |     | 
C |  G4 |     |     |     |     |     |     |     |     |     |     |     |    
D    |  P4 |     |     |     |     |     |     |     |     |     |     |     | 
E |     |  O4 |     |     |     |     |     |     |     |     |     |     |    
F    |     |  Y4 |     |     |     |     |     |     |     |     |     |     | 
G |     |     |  R4 |     |     |     |     |     |     |     |     |     |    
H    |     |     |     |     |     |     |     |     |     |     |     |     | 
I |     |     |     |     |     |     |     |     |     |     |     |     |    
J    |     |     |     |     |     |     |     |     |     |     |     |     | 
K |     |     |     |     |     |     |     |     |     |     |     |     |    
L    |     |     |     |     |     |     |     |     |     |     |     |     | 
M |     |     |     |     |     |     |     |     |     |     |     |     |    
N    |     |     |     |     |     |     |     |     |     |     |     |     | 
O |     |     |     |     |     |     |     |     |     |     |     |     |    
P    |     |     |     |     |     |     |     |     |     |     |     |     | 
Q |     |     |     |     |     |     |     |     |     |     |     |     |    
R    |     |     |     |     |     |     |     |     |     |     |     |     | 
S |     |     |     |     |     |     |     |     |     |     |     |     |    
T    |     |     |     |     |     |     |     |     |     |     |     |     | 
U |     |     |     |     |     |     |     |     |     |     |     |     |    
V    |     |     |     |     |     |     |     |     |     |     |     |     | 
W |     |     |     |     |     |     |     |     |     |     |     |     |    
X    |     |     |     |     |     |     |     |     |     |     |     |     | 
Y |     |     |     |     |     |     |     |     |     |     |     |     |    
Z    |     |     |     |     |     |     |     |     |     |     |     |     | 
  -----------------------------------------------------------------------------

Your hand is...
Bag Size: 6
P3, P2, G6, B6, G2, B3
> save
Please enter a filename
>test01
Game Successfully Saved

SHREY PAREKH'S turn.
SHREY PAREKH score: 9
JACK SWALLOW score: 18
  0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
  -----------------------------------------------------------------------------
A |     |     |     |     |     |     |     |     |     |     |     |     |    
B B4 |     |     |     |     |     |     |     |     |     |     |     |     | 
C |  G4 |     |     |     |     |     |     |     |     |     |     |     |    
D    |  P4 |     |     |     |     |     |     |     |     |     |     |     | 
E |     |  O4 |     |     |     |     |     |     |     |     |     |     |    
F    |     |  Y4 |     |     |     |     |     |     |     |     |     |     | 
G |     |     |  R4 |     |     |     |     |     |     |     |     |     |    
H    |     |     |     |     |     |     |     |     |     |     |     |     | 
I |     |     |     |     |     |     |     |     |     |     |     |     |    
J    |     |     |     |     |     |     |     |     |     |     |     |     | 
K |     |     |     |     |     |     |     |     |     |     |     |     |    
L    |     |     |     |     |     |     |     |     |     |     |     |     | 
M |     |     |     |     |     |     |     |     |     |     |     |     |    
N    |     |     |     |     |     |     |     |     |     |     |     |     | 
O |     |     |     |     |     |     |     |     |     |     |     |     |    
P    |     |     |     |     |     |     |     |     |     |     |     |     | 
Q |     |     |     |     |     |     |     |     |     |     |     |     |    
R    |     |     |     |     |     |     |     |     |     |     |     |     | 
S |     |     |     |     |     |     |     |     |     |     |     |     |    
T    |     |     |     |     |     |     |     |     |     |     |     |     | 
U |     |     |     |     |     |     |     |     |     |     |     |     |    
V    |     |     |     |     |     |     |     |     |     |     |     |     | 
W |     |     |     |     |     |     |     |     |     |     |     |     |    
X    |     |     |     |     |     |     |     |     |     |     |     |     | 
Y |     |     |     |     |     |     |     |     |     |     |     |     |    
Z    |     |     |     |     |     |     |     |     |     |     |     |     | 
  -----------------------------------------------------------------------------

Your hand is...
Bag Size: 6
P3, P2, G6, B6, G2, B3
>quit 
Goodbye!

